app/ui/src/app/customizations/api-connector/api-connector-list.component.ts

```diff
@@ -78,5 +83,14 @@ export class ApiConnectorListComponent implements OnInit {
   ngOnInit() {
     this.appName = this.config.getSettings('branding', 'appName', 'Syndesis');
     this.apiConnectorStore.loadAll();
+
+    const apiConnectorState$ = this.apiStore.select('apiConnectorState');
+    apiConnectorState$.subscribe(apiConnectorState => {
+      console.log(apiConnectorState);
+    });
+    const metadataState$ = this.store.select('metadataState');
+    metadataState$.subscribe(metadataState => {
+      console.log(metadataState);
+    });
```

Comment:
I've dropped this for demo purposes whilst this app is still a WIP, so the UI team can see how we access to slices of state from within our components.